### PR TITLE
Added heartbeats plug in and CustomCheck sample

### PR DESCRIPTION
**Pull request by seanfarmar**

### CODE REVIEW

**Comment by mikeminutillo** [comment]
@seanfarmar what prompted this PR? Are you using this sample to demo stuff or was there an issue with the demo as is?

**Comment by seanfarmar** [comment]
I'm using this as a demo for the platform and I was missing this...

**Comment by seanfarmar** [comment]
I've updated this with v7 and updated ServiceControl and ServicePulse.

Happy to re PR this if you want separate PRs for each update?

File: Platform/servicecontrol/monitoring-instance/.diagnostics/Particular.Monitoring-configuration.txt [hunk 1]
```diff
@@ -0,0 +1 @@
...criminator":null,"Properties":{"Schema":"dbo","Catalog":"ParticularMonitoringDemo"}}},"PurgeOnStartup":false,"QueueNameBase":"Particular.Monitoring","TransactionMode":"ReceiveOnly","MaxConcurrency":4,"Satellites":[{"Name":"Timeout Message Processor","ReceiveAddress":"Particular.Monitoring.Timeouts@[dbo]@[ParticularMonitoringDemo]","TransactionMode":"ReceiveOnly","MaxConcurrency":4},{"Name":"Timeout Dispatcher Processor","ReceiveAddress":"Particular.Monitoring.TimeoutsDispatcher@[dbo]@[ParticularMonitoringDemo]","TransactionMode":"ReceiveOnly","MaxConcurrency":4}]},"Recoverability":{"ImmediateRetries":5,"DelayedRetries":3,"DelayedRetriesTimeIncrease":"0:00:10","ErrorQueue":"error","UnrecoverableExceptions":["NServiceBus.MessageDeserializationException"]},"Serialization":{"DefaultSerializer":{"Type":"NServiceBus.NewtonsoftSerializer","Version":"2.1.0","ContentType":"application/json"},"AdditionalDeserializers":[{"Type":"ServiceControl.Monitoring.Messaging.TaggedLongValueWriterOccurrenceSerializerDefinition","Version":"2.0.0","ContentType":"TaggedLongValueWriterOccurrence"}]},"Transport":{"Type":"ServiceControl.Transports.SQLServer.ServiceControlSQLServerTransport","Version":"0.0.0"}}
```
Comment (mikeminutillo) [inline Platform/servicecontrol/monitoring-instance/.diagnostics/Particular.Monitoring-configuration.txt]: This folder should be ignored I think. We don't want to check in the diagnostics

**Comment by tmasternak** [comment]
@seanfarmar this is what I get running the pakackage:

``` PS 
Updating connection strings
Get-Content : Cannot find path 'C:\git\p\MonitoringDemo\Solution\binaries\CustomCheck\net461\CustomCheck.exe.config'
because it does not exist.
At C:\git\p\MonitoringDemo\localDB.ps1:69 char:16
+   $xml = [xml](Get-Content $ConfigFile)
+                ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\git\p\Monito...heck.exe.config:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

C:\git\p\MonitoringDemo\localDB.ps1 : Error starting setting up demo.
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,localDB.ps1

    + FullyQualifiedErrorId : {4}

 :
At C:\git\p\MonitoringDemo\localDB.ps1:70 char:3
+   $xml.SelectNodes("//connectionStrings/add") | % {
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
```

The build output for CustomCheck is missing from the repo - I think we need it.

Secondly I would propose to merge `3rdPartySystem` and `CustomCheck` into single project. Having two separate adds complexity to the project that users need to analyze and we only control the 3rdPartySystem http codes so single console for interaction should be enough.

**Comment by seanfarmar** [comment]
@mikeminutillo @tmasternak I'll review this next week and fix, thanks

File: Platform/servicecontrol/monitoring-instance/.diagnostics/Particular.Monitoring-configuration.txt (hunk 1 above)
Comment (seanfarmar) [reply Platform/servicecontrol/monitoring-instance/.diagnostics/Particular.Monitoring-configuration.txt]: pushed a fix

**Comment by seanfarmar** [comment]
In the last push, I've moved custom checks to Shipping and removed the customCheck project (I'm hoping to add 3rd party integration on shipping to show saga monitoring). 
I also removed the .diagnostics folder form committed files

@tmasternak @mikeminutillo what do you think?

**Comment by danielmarbach** [comment]
It is probably better to close this PR and restart with a clean slate given that so much has changed since

**Pull request closed without merging**